profiler/src/ProfilerEngine/Datadog.Profiler.Native/RawThreadLifetimeSample.h

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+
+#pragma once
+
+#include "RawSample.h"
+
+class SampleValueTypeProvider;
+
+enum ThreadEventKind
```

Comment:
prefer `enum class`, is there a reason why not using it?

Reply:
... only the force of the habits  :^)
✅ done